core/templates/dev/head/player/exploration_player.html

```diff
@@ -105,6 +107,83 @@
     <feedback-popup></feedback-popup>
   </script>
 
+  <script type="text/ng-template" id="popover/informationCard">
+    <md-card class="md-padding">
+      <div ng-style="informationCardBackgroundCcs" class="information-card-logo-thumbnail">
+        <h2 class="information-card-title"><[explorationCardTitle]></h2>
+      </div>
+      <div class="information-card-content">
+        <p>The titles of Washed Outs breakthrough song and the first single from Paracosm share the two most important.</p>
+        <ul layout="row" class="card-metrics" layout-align="space-between center">
+          <li><span class="fa fa-star fa-lg"></span>
+            <[explorationRatings]>
+          </li>
+          <li><span class="fa fa-eye fa-lg"></span>
+            <[viewsCount]>
+          </li>
+          <li><span class="fa fa-clock-o fa-lg"></span>
+            <[lastModifiedDate | date: 'MM/dd/yyyy']>
+          </li>
+          <li layout="row" layout-align="space-between center">
+            <img src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
+            <ul style="padding:5px;" layout="row">
+              <li><img class="img-circle" ng-src="/images/general/user_blue_72px.png" alt=""></li>
+              <li><img class="img-circle" ng-src="/images/general/user_mint_48px.png" alt=""></li>
+              <li ng-if='explorationContributors.length > 2' class="circle">
+                <[explorationContributors.length - 2]>+</li>
+            </ul>
+          </li>
+        </ul>
+        <div class="social-media-and-tags" layout="row">
+          <div flex="55">
+            <md-content layout="row">
+              <div class="oppia-tag-icon"><img src="/third_party/static/material-design-icons-1.0.1/ic_tag.png"></div>
+              <md-chips>
```

Comment:
Something seems a bit off with the indentation and the number of opening/closing tags for all these md-chip(s).